dist/realtime.js

```diff
@@ -693,31 +688,52 @@
      * Triggering
      */
 
-    on: function (name, handler) {
-      if (this.__bindings.hasOwnProperty(name))
-        this.__bindings[name].push(handler);
-      else
-        this.__bindings[name] = [handler];
+    on: function (name, handler, context, once) {
+      if (context === null)
+        context = undefined;
+      if (!this.__bindings.hasOwnProperty(name))
+        this.__bindings[name] = [];
+      this.__bindings[name] = this.__bindings[name].concat([
+        [handler, context, Boolean(once)]
+      ]);
     },
 
-    off: function (name, handler) {
+    off: function (name, handler, context) {
+      if (context === null)
+        context = undefined;
       if (!this.__bindings.hasOwnProperty(name))
         return;
-      var index = $.inArray(handler, this.__bindings[name]);
-      if (index !== -1)
-        this.__bindings[name].splice(index, 1);
+      this.__bindings[name] = this.__bindings[name].filter(function (listener) {
```

Comment:
this vendor won't work for any project not including neither Core.js nor any other polyfill on IE < 9.